Lifecycle.md

```diff
@@ -0,0 +1,279 @@
+# Planetary Software Organization Project and Working Group Lifecycle
+
+## Definitions
+
+The Planetary Software Organization hosts several Top-Level Projects
+(TLPs). These projects are autonomous from each other and governed
+by their own TC (Technical Committee) and chartered by the Planetary
+Software TSC.  Typically, TLPs are software projects (but not always)
+and are typically long-lived and open-ended.
+
+The Planetary Software Organization also hosts several Top-Level Working
+Groups (TLWGs).  TLWGs are typically formed for a specific purpose,
+and when that purpose is achieved, the TLWG will stand down.
+
+TLPs are free to create their own Working Groups which are autonomous groups
+collaborating to fulfill a set of responsibilities. Working Groups are
+eventually chartered by their project TC.
```

Comment:
I would change the last sentence to `These Working Groups are charted by the TC of the project they exist under.`

Reply:
Fixed.